pandas/io/tests/test_packers.py

```diff
@@ -795,18 +795,19 @@ class TestMsgpack():
     http://stackoverflow.com/questions/6689537/nose-test-generators-inside-class
     """
 
-    def setUp(self):
```

Comment:
Pytest will run `setup` but not `setUp` before each test in a class.  Guessing this change in structure was intentional.